src/components/DNSenumTool/DNSenumTool.tsx

```diff
@@ -54,7 +56,7 @@ const DnsenumTool = () => {
         "Step 7: Select Reverse Lookup if required.\n";
     const sourceLink = "https://www.kali.org/tools/dnsenum/";
     const tutorial = "https://www.kali.org/tools/dnsenum/";
-    const dependencies = ["dnsenum"]; // Dependencies required by the component
+    const dependencies = ["dnsenum", "kill"]; // Dependencies required by the component
```

Comment:
Should not require kill
```suggestion
    const dependencies = ["dnsenum"]; // Dependencies required by the component
```